### PR TITLE
DDP-4395 fix downloading pdfs for consent suspended participants

### PR DESCRIPTION
**Pull request by yufengwng**
## Context

Fixes DDP-4395.

Root cause seems to be that we're strictly checking that the status is `enrolled` before allowing to download pdf. This PR changes the check so that if we have ever been `enrolled` in the past then we allow downloading pdf.

## Checklist

- [x] I have labeled the type of changes involved using the `C-*` labels.
- [x] I have assessed potential risks and labeled using the `R-*` labels.
- [x] I have considered error handling and alerts, and added `L-*` labels as needed.
- [x] I have considered security and privacy, and added `I-*` labels as needed
- [x] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [x] (n/a) If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [x] (n/a) If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

_If unsure or need help with any of the above items, add the `help wanted` label. For items that starts with `If applicable`, if it is not applicable, check it off and add `n/a` in front._

## FUD Score

_Overall, how are you feeling about these changes?_

- [x] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [x] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [x] I have written automated positive tests
- [ ] I have written automated negative tests
- [x] I have poked around locally to verify proper functionality
- [ ] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [x] These changes require no special release procedures--just code!
- [ ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

**Comment by yufengwng** [comment]
please run tests

File: pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiUserStudyEnrollment.java [hunk 1]
```diff
@@ -196,9 +196,10 @@ int deleteByUserGuidStudyGuid(@Bind("userGuid") String userGuid,
                     + " JOIN umbrella_study us ON us.umbrella_study_id = usen.study_id "
                     + " AND usen.user_id = :userId"
                     + " AND usen.study_id = :studyId"
+                    + " ORDER BY usen.valid_from desc"
```
Comment (ssettipalli) [inline pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiUserStudyEnrollment.java:199]: Just curious: Looks like we are streaming through all status rows to fond if enrolled before or not.
Are we gaining much with this sort !

**Review by ssettipalli — APPROVED** [review]
looks good to me

**Review by raiks2 — APPROVED** [review]
LGTM

File: pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiUserStudyEnrollment.java (hunk 1 above)
Comment (yufengwng) [reply pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiUserStudyEnrollment.java:199]: We are streaming, but this sort here means we don't have to do a sort on the stream to get the latest enrollment status.

**Comment by yufengwng** [comment]
please run tests

**Pull request merged**